rasa/core/processor.py

```diff
@@ -420,17 +420,23 @@ def _log_slots(tracker) -> None:
             logger.debug(f"Current slot values: \n{slot_values}")
 
     def _log_unseen_features(self, parse_data: Dict[Text, Any]) -> None:
-        """Check if the NLU interpreter picks up intents or entities that aren't
-        recognized."""
+        """Logs unrecognized features from the NLU parse data.
```

Comment:
"Log" is a little weird, because nothing is actually logged. Maybe we should rename it to `_check_for_unseen_features` and adjust the doc string accordingly.

Reply:
oh true, it doesn't log but actually raises warnings! I think this is leftover from when we used `logger.warning`, good catch :)